cmd/estuary-shuttle/main.go

```diff
@@ -223,22 +227,27 @@ func main() {
 		},
 		&cli.BoolFlag{
 			Name:  "logging",
+			Usage: "enable api endpoint logging",
 			Value: cfg.LoggingConfig.ApiEndpointLogging,
 		},
 		&cli.BoolFlag{
 			Name:  "write-log-flush",
+			Usage: "enable hard flushing blockstore",
```

Comment:
Maybe we could make this wording better? I'm not sure what it means just by looking at it